dbt_subprojects/dex/seeds/trades/_schema.yml

```diff
@@ -5221,6 +5221,21 @@ seeds:
         block_number: uint256
         token_bought_address: varbinary
         token_sold_address: varbinary
-        token_bought_amount_raw: uint256
-        token_sold_amount_raw: uint256
+        token_bought_amount_raw: decimal(38,0)
+        token_sold_amount_raw: decimal(38,0)
+        block_date: timestamp
```

Comment:
Is it possible to make `token_bought_amount_raw` and `token_sold_amount_raw` uint256 type?